scripts/generate_json_docs.py

```diff
@@ -259,14 +255,14 @@ def build_link_from_type(type_name, object_type=None):
     return type_markup
 
 
-def build_source(mod, method):
-    if isinstance(mod, (types.ModuleType, types.ClassType,
-                        types.MethodType, types.FunctionType,
-                        types.TracebackType, types.FrameType,
-                        types.CodeType, types.TypeType)):
+def build_source(module, method):
+    if isinstance(module, (types.ModuleType, types.ClassType,
+                           types.MethodType, types.FunctionType,
+                           types.TracebackType, types.FrameType,
+                           types.CodeType, types.TypeType)):
 
-        line = inspect.getsourcelines(mod)[1]
-        source_path = clean_source_path(inspect.getsourcefile(mod))
+        line = inspect.getsourcelines(module)[1]
```

Comment:
What does `inspect.getsourcelines` return? You should assert it by unpacking some `_` as well (pending the size). e.g.

```python
_, line = inspect.getsourcelines(module)
# OR
unused_content, line = inspect.getsourcelines(module)
```

A